includes/admin/payments/orders.php

```diff
@@ -889,6 +889,7 @@ function edd_order_details_attributes( $order ) {
 						<?php echo esc_html( __( 'Time', 'easy-digital-downloads' ) . ' (' . edd_get_timezone_abbr() . ')' ); ?>
 					</label>
 
+					<label for="edd_payment_time_hour_chosen" class="screen-reader-text">Hour</label>
```

Comment:
This has the same issue as the add download select--`chosen` should not be included in the label, as it's not the ID of the actual `select` element. Looks like this `for` should be `edd_payment_time_hour`, as the `id` parameter being passed to the `EDD()->html->select()` method is being disregarded.